### PR TITLE
[GLUTEN-5211][VL] Fix typos in velox-backend-support-progress.md

### PR DESCRIPTION
**Pull request by xumingming**
1. Some wording polishing.
2. Ansi Off -> ANSI OFF
3. Draw -> Drag

### CODE REVIEW

File: docs/velox-backend-support-progress.md [hunk 1]
```diff
@@ -20,18 +24,20 @@ The total supported functions' number for [Spark3.3 is 387](https://spark.apache
...
 
 
+And also some notations for the function implementation's restrictions:
+
 | Value      | Description                                                                                                                                                      |
-| ---------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Mismatched | Some functions are implemented by Velox, which return results mismatched with Apache Spark. So we marked then as "Mismatched".                                   |
-| Ansi OFF   | Gluten doesn't support [ANSI mode](https://spark.apache.org/docs/latest/sql-ref-ansi-compliance.html). If it is enabled, Gluten will fall back to Vanilla Spark. |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Mismatched | Some functions are implemented by Velox, but have different semantics from Apache Spark, we mark them as "Mismatched".                                           |
```
Comment (yma11) [inline docs/velox-backend-support-progress.md:31]: `Some functions are implemented by Velox for Presto, but have different semantics from Apache Spark`?

File: docs/velox-backend-support-progress.md [hunk 2]
```diff
@@ -202,7 +208,7 @@ Gluten supports 199 functions. (Draw to right to see all data types)
 | regexp_like                   | regexp_like            | rlike                 | S      | Not support lookaround |         |      |       |     |      |       |        |      |           | S      |         |      |        |          |       |      |        |      |
 | regexp_replace                | regexp_replace         |                       | S      |                        |         |      |       |     |      |       |        |      |           | S      |         |      |        |          |       |      |        |      |
 | rlike                         |                        | rlike                 | S      | Not support lookaround |         |      |       |     |      |       |        |      |           | S      |         |      |        |          |       |      |        |      |
```
Comment (yma11) [inline docs/velox-backend-support-progress.md:210]: `lookaround not supported`?

**Comment by yma11** [comment]
Thanks for updating this.

File: docs/velox-backend-support-progress.md (hunk 1 above)
Comment (PHILO-HE) [reply docs/velox-backend-support-progress.md:31]: @yma11, it is still possible that some velox functions implemented for Spark have semantic difference from Apache Spark. So I think it's good to use the PR-proposed statement.

**Review by PHILO-HE — APPROVED** [review]
Thanks!

File: docs/velox-backend-support-progress.md (hunk 1 above)
Comment (yma11) [reply docs/velox-backend-support-progress.md:31]: > it is still possible that some velox functions implemented for Spark have semantic difference from Apache Spark.

Shouldn't this be identified as bug that we should fix? We'd better not allow this case.

File: docs/velox-backend-support-progress.md (hunk 2 above)
Comment (PHILO-HE) [reply docs/velox-backend-support-progress.md:210]: @yma11, yes, this is a known limit of RE2.

File: docs/velox-backend-support-progress.md (hunk 1 above)
Comment (PHILO-HE) [reply docs/velox-backend-support-progress.md:31]: For some cases, we need to fix. But for some other cases, it is not easy for us to fix. For example, in regexp functions, when to use "\\\s" to match space, vanilla Spark treats "\v" as space, but RE2 (used by velox's regexp functions) doesn't, which is a kind of semantic difference in third-party lib.

**Review by PHILO-HE — APPROVED** [review]
Thanks! Will merge it soon if no other comment. cc @yma11

**Pull request merged**